docs/tutorials/buttons.rst

```diff
@@ -159,31 +168,35 @@ pleasant game, so perhaps you can figure out how to improve it.)::
 
     display.clear()
 
-Can you see how we check what buttons are pressed? We used ``if``,
-``elif`` (short for "else if") and ``else``. These are called *conditionals*
-and work like this::
+Czy widzisz jak sprawdzamy jakie przyciski są wciśnięte? Użyliśmy ``if`` (ang. jeśli),
+``elif`` (skrót od "else if") (ang. a jednak jeśli) oraz ``else`` (ang. inaczej).
```

Comment:
Dziwnie brzmi tłumaczenie if/else w polskim języku, ale może dlatego że nie spotkałem się z nim do tej pory. Sprawdzałeś może jak to się tłumaczy w fachowej literaturze?

Reply:
Nie znalazłem, trochę szukałem i nic konkretnego. Sądzę, że może będzie to w jakichś książkach do podstaw Pythona. Tym czasem mam takie propozycje: "else if" to "jeśli jednak" oraz "else" to "w pozostałych przypadkach".